cmd/vclusterctl/cmd/platform/add/vcluster.go

```diff
@@ -33,7 +33,7 @@ vcluster platform add vcluster my-vcluster --namespace vcluster-my-vcluster --pr
 	`
 
 	addCmd := &cobra.Command{
-		Use:   "vcluster",
+		Use:   "vcluster VCLUSTER_NAME",
```

Comment:
We could also use this util func that we already use in some places (probably not consistent over the whole code base): https://github.com/loft-sh/vcluster/blob/main/cmd/vclusterctl/cmd/create.go#L35